### DISCUSSION TITLE
Simulating patients with multiple conditions

### DISCUSSION THREAD

**Discussion by r0sies** [synthetichealth/synthea#1265] [Q&A] [answered]
Hi,
I would like to calculate risk-adjusted mortality rates for acute myocardial infarction.
How do I generate populations of people who have had acute myocardial infarction as well as other conditions (pre-conditions/comorbidities) from a set of 10 conditions (including for exampl, dementia, kidney failure, malignancy etc)
I can use the Myocardial Infarction module as a base, but how do I force people to go through a set of other disease modules?
Thanks!

**Comment by jawalonoski** [comment, marked as answer]
You have a few options.

1. Generate 10K patients or more, and then search the output (using a script you write) to find patients who meet your criteria. Use only those patients.
2. Write a [Keep Module](https://github.com/synthetichealth/synthea/wiki/Keep-Patients-Module) that specifies the criteria you want. This will essentially be the same results as 1) above, except Synthea will not export patients who do not meet your criteria.
3. Edit the disease modules of interest using the [Module Builder](https://synthetichealth.github.io/module-builder/) to increase prevalence rates to ensure each patient has the conditions you want.
4. Write a [Module Overrides File](https://github.com/synthetichealth/synthea/wiki/Module-Overrides) that overrides the transition probabilities of modules during runtime. This is equivalent to 4) above.

I typically recommend option 1 or 2.

Option 3 or 4 should only be used if you are well-verse with the software modules, because editing the modules can have side-effects if you don't know what you are doing.

**Reply by r0sies** [reply]
Thanks, within a module I increased the prevalence for conditions. 
Can I force a single patient to go through the myocardial infarction module, congestive heart failure, and the dementia module? or is it just option 1, where you hope a large enough population will have hundreds of patients that will go through _all three_ modules.

**Reply by jawalonoski** [reply]
You can use the Keep Module to filter out patients who do or do not go through those modules, but you can't "force" them unless you use 3) or 4).

**Reply by r0sies** [reply]
I see, great, thank you for clarifying.
